README.md

```diff
@@ -1,29 +1,27 @@
 
-# Jellyfish iOS Client
-
-[Jellyfish](https://github.com/jellyfish-dev/jellyfish) Client library for iOS apps written in Swift.
+# Fishjam iOS Client
+[Fishjam](https://github.com/fishjam-dev/fishjam) Client library for iOS apps written in Swift.
 
 ## Components
 
 The repository consists of 2 separapable components:
 
-- `JellyfishClientSdk` - Jellyfish client fully compatible with `Jellyfish`, responsible for exchaning media events and receiving media streams which then are presented to the user
-- `JellyfishCLientDemo` - Demo application utilizing `Jellyfish` client
+- `FishjamClientSdk` - Fishjam client fully compatible with `Fishjam`, responsible for exchaning media events and receiving media streams which then are presented to the user
+- `FishjamCLientDemo` - Demo application utilizing `Fishjam` client
```

Comment:
Right now we don't have `FishjamClientSdk` and `FishjamCLientDemo`. I'm not sure whether we should change it now

Reply:
U right.

Reply:
Gone back. It will be changed later.